pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -149,24 +126,23 @@ def work(self):
                                     'CATCH_POKEMON']['status']
                                 if status is 2:
                                     logger.log(
-                                        '[-] Attempted to capture {} - failed.. trying again!'.format(pokemon_name), 'red')
+                                        '[-] Attempted to capture {}- failed.. trying again!'.format(pokemon_name), 'red')
                                     sleep(2)
                                     continue
                                 if status is 3:
                                     logger.log(
                                         '[x] Oh no! {} vanished! :('.format(pokemon_name), 'red')
                                 if status is 1:
-                                    
+                                    logger.log(
+                                        '[x] Captured {}! [CP {}] [IV {}]'.format(
+                                            pokemon_name,
+                                            cp,
+                                            pokemon_potential
+                                        ), 'green'
+                                    )
+
                                     id_list2 = self.count_pokemon_inventory()
-                                    
```

Comment:
you deleted changes on Captured pokemons messages that are not related to your PR